scipy/special/cython_special.pxd

```diff
@@ -190,7 +190,7 @@ cpdef double nbdtrc(dlp_number_t x0, dlp_number_t x1, double x2) noexcept nogil
 cpdef double nbdtri(dlp_number_t x0, dlp_number_t x1, double x2) noexcept nogil
 cpdef double nbdtrik(double x0, double x1, double x2) noexcept nogil
 cpdef double nbdtrin(double x0, double x1, double x2) noexcept nogil
-cpdef double ncfdtr(double x0, double x1, double x2, double x3) noexcept nogil
+cpdef df_number_t ncfdtr(df_number_t x0, df_number_t x1, df_number_t x2, df_number_t x3) noexcept nogil
```

Comment:
I don't know what this means, but I'm happy to assume it is the right thing.

Reply:
Fused type in Cython. Let’s the function take both double or float and dispatch to the right one. It’s like a bootleg way of handling what templates can do in C++.